lib/r10k/api/modules.rb

```diff
@@ -123,12 +125,17 @@ def write_module(module_name, env_map, path, opts={})
             git.clean(path, git_dir: git_dir, force: true)
           end
         when :forge
-          tarball_cachedir = cachedir_for_forge_module(mod[:source], opts[:cachedir])
+          release_slug = release_slug_from_module_slug_version(mod[:source], mod[:resolved_version])
+          cachedir = opts[:cachedir]
 
-          # TODO: cache tarball
+          unpack_tmpdir = cachedir_for_forge_module([release_slug, 'unpack'].join('-'), cachedir)
```

Comment:
perhaps this should just be `join(cachedir, unpack)` or `join(cachedir,unpack,release_slug)`. This shouldn't be considered API so I guess it doesn't block the merge.